spring-cloud-kubernetes-integration-tests/spring-cloud-kubernetes-fabric8-client-configmap-event-reload/src/test/resources/logback-test.xml

```diff
@@ -11,4 +11,5 @@
 
 	<logger name="org.testcontainers" level="INFO"/>
 	<logger name="com.github.dockerjava" level="WARN"/>
+	<logger name="io.fabric8.kubernetes.client" level="ERROR"/>
```

Comment:
fabric8 okhttp client logs too much, on already lots of logs that we have in integration tests...